plugin/fzf.vim

```diff
@@ -506,7 +518,7 @@ function! s:execute(dict, command, use_height, temps) abort
   endif
   if s:is_win
     let batchfile = s:fzf_tempname().'.bat'
```

Comment:
Let's add it to `temps` then.
The code change should be a minimal (one-liner?) so can you commit the fix?
I'd rather not handle it in this pull request.